### PR TITLE
Replace legacy ingest with raw query operation

### PR DESCRIPTION
**Pull request by sergiimk**

### CODE REVIEW

File: src/main/scala/dev/kamu/engine/spark/RawQuery.scala [hunk 1]
```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2018 kamu.dev
```
Comment (s373r) [inline src/main/scala/dev/kamu/engine/spark/RawQuery.scala:2]: General PR comment: old copyright headers

File: adapter/src/adapter.rs [hunk 2]
```diff
@@ -15,7 +15,7 @@ use crate::spark::forward_to_logging;
 #[derive(Debug)]
 pub struct SparkODFAdapter {
     // spark_master: SparkMaster,
-// spark_worker: SparkWorker,
+    // spark_worker: SparkWorker,
```
Comment (s373r) [inline adapter/src/adapter.rs:18]: A file comment: could you please add a comment re commented code?

... or if it isn't needed, please delete it

File: adapter/src/grpc.rs [hunk 3]
```diff
@@ -51,22 +52,56 @@ impl EngineGRPC for EngineGRPCImpl {
...
 
             tx.send(Ok(response_grpc)).await.unwrap();
 
             // let responses = [
-            //     ExecuteQueryResponse::Progress,
-            //     ExecuteQueryResponse::Progress,
-            //     ExecuteQueryResponse::Error,
+            //     TransformResponse::Progress,
+            //     TransformResponse::Progress,
+            //     TransformResponse::Error,
```
Comment (s373r) [inline adapter/src/grpc.rs:104]: A file comment: could you please add a comment re commented code?

... or if it isn't needed, please delete it

**Review by s373r — CHANGES_REQUESTED** [review]

File: src/main/scala/dev/kamu/engine/spark/RawQuery.scala (hunk 1 above)
Comment (sergiimk) [reply src/main/scala/dev/kamu/engine/spark/RawQuery.scala:2]: yeah, this repo needs some love...

**Pull request merged**